config/mapfiles.d/50-central-collector.conf

```diff
@@ -1,11 +0,0 @@
-###############################################################################
-#
-# HTCondor-CE Central Collector default authentication mappings
-#
-# DO NOT EDIT THIS FILE!  It will be overwritten upon RPM upgrade.
-# If you wish to make changes to the HTCondor-CE mappings, create files
-# in /etc/condor-ce/mapfiles.d containing your changes.
-#
-###############################################################################
-
-GSI /[.A-Za-z0-9\/= -]*\/CN=([.A-Za-z0-9\/= -]+)/ \1@unmapped.htcondor.org
```

Comment:
If GSI does get used, do unmapped users get mapped to `gsi@unmapped`?

Reply:
If HTCondor 9.0.0 is being used, then yes. I hope we can assume a more recent version of HTCondor at this point.

Reply:
Yeah, you guys don't support anything older so I think this is kosher!